packages/static-wado-deploy/lib/retrieveDeploy.mjs

```diff
@@ -0,0 +1,23 @@
+import DeployGroup from "./DeployGroup.mjs";
+
+export async function retrieveF(retrieveDirectory, config, name, options, deployPlugin) {
```

Comment:
If I got correct your implementation I believe there is no need to export this method. 
It can be private to module scope instead and be used directly on other function

Reply:
Now I only export that method, as the other one is identical to the common code.
Also, the bind is required because I needed an extra parameter provided to the retrieve function, and the retrieve/store functions are used elsewhere.